eratos_xarray/backend/eratos_.py

```diff
@@ -103,8 +111,20 @@ def __init__(self, gsdata):
         self.gsdata = gsdata
 
     @classmethod
-    def open(cls, ern, eratos_auth: AccessTokenCreds = None):
-        adapter = Adapter(eratos_auth)
+    def open(
+        cls,
+        ern,
+        eratos_auth: Optional[BaseCreds] = None,
+        ignore_certs: bool = False,
+        eratos_adapter: Optional[Adapter] = None,
+    ):
+        if eratos_adapter:
+            adapter = eratos_adapter
+        elif eratos_auth:
+            print(f"ignore_certs: {ignore_certs}")
+            adapter = Adapter(eratos_auth, ignore_certs=ignore_certs)
+        else:
+            raise ValueError("Need Eratos creds or adapter object")
```

Comment:
I think the message can be more specific...

```suggestion
            raise ValueError("No Eratos credentials found.  Pass credentials explicitly via BaseCreds or Adapter objects'")
```